serial/uart.c

```diff
@@ -81,12 +130,19 @@ uart_config(uint8 uart_no)
     // to set the threshold here...
     // We do not enable framing error interrupts 'cause they tend to cause an interrupt avalanche
     // and instead just poll for them when we get a std RX interrupt.
+
+	uint32_t tx_empty_bits = 0;
+	if (uart0_tx_enable_pin >= 0) {
+	  tx_empty_bits = (0 & UART_TXFIFO_EMPTY_THRHD) << UART_TXFIFO_EMPTY_THRHD_S | UART_TXFIFO_EMPTY_INT_ENA;
```

Comment:
It's UART_TXFIFO_EMPTY_INT_ENA, the first part evaluates to 0, but I left it there to show that we are setting thethreshold to 0.